Files already downloaded and verified
Files already downloaded and verified
Running Evolution for fitness
Initializing
Testing: 0it [00:00, ?it/s]
  0%|                                                                                            | 0/5 [00:00<?, ?it/s]


















  0%|                                                                                            | 0/5 [00:48<?, ?it/s]
Traceback (most recent call last):
  File "C:\Users\Jordan\Learning\UVM\Research\novel-feature-detector\evolution.py", line 242, in <module>
    run()
  File "C:\Users\Jordan\Learning\UVM\Research\novel-feature-detector\evolution.py", line 197, in run
    solution_over_time, fitness_over_time = evolution(generations=n_iters, population_size=pop_size, num_children=num_children, tournament_size=tournament_size, num_winners=num_winners, evolution_type=run_name)
  File "C:\Users\Jordan\Learning\UVM\Research\novel-feature-detector\evolution.py", line 92, in evolution
    trainer.test(net, dataloaders=data_module.val_dataloader())
  File "C:\Users\Jordan\anaconda3\envs\EC2\lib\site-packages\pytorch_lightning\trainer\trainer.py", line 939, in test
    return self._call_and_handle_interrupt(self._test_impl, model, dataloaders, ckpt_path, verbose, datamodule)
  File "C:\Users\Jordan\anaconda3\envs\EC2\lib\site-packages\pytorch_lightning\trainer\trainer.py", line 724, in _call_and_handle_interrupt
    return trainer_fn(*args, **kwargs)
  File "C:\Users\Jordan\anaconda3\envs\EC2\lib\site-packages\pytorch_lightning\trainer\trainer.py", line 986, in _test_impl
    results = self._run(model, ckpt_path=self.ckpt_path)
  File "C:\Users\Jordan\anaconda3\envs\EC2\lib\site-packages\pytorch_lightning\trainer\trainer.py", line 1237, in _run
    results = self._run_stage()
  File "C:\Users\Jordan\anaconda3\envs\EC2\lib\site-packages\pytorch_lightning\trainer\trainer.py", line 1321, in _run_stage
    return self._run_evaluate()
  File "C:\Users\Jordan\anaconda3\envs\EC2\lib\site-packages\pytorch_lightning\trainer\trainer.py", line 1366, in _run_evaluate
    eval_loop_results = self._evaluation_loop.run()
  File "C:\Users\Jordan\anaconda3\envs\EC2\lib\site-packages\pytorch_lightning\loops\base.py", line 211, in run
    output = self.on_run_end()
  File "C:\Users\Jordan\anaconda3\envs\EC2\lib\site-packages\pytorch_lightning\loops\dataloader\evaluation_loop.py", line 187, in on_run_end
    self._evaluation_epoch_end(self._outputs)
  File "C:\Users\Jordan\anaconda3\envs\EC2\lib\site-packages\pytorch_lightning\loops\dataloader\evaluation_loop.py", line 307, in _evaluation_epoch_end
    self.trainer._call_lightning_module_hook("test_epoch_end", output_or_outputs)
  File "C:\Users\Jordan\anaconda3\envs\EC2\lib\site-packages\pytorch_lightning\trainer\trainer.py", line 1596, in _call_lightning_module_hook
    output = fn(*args, **kwargs)
  File "C:\Users\Jordan\Learning\UVM\Research\novel-feature-detector\net.py", line 215, in test_epoch_end
    for k, v in x['val_class_acc'].items():
KeyError: 'val_class_acc'